docs/compliance/cis/k8s.md

```diff
@@ -4,8 +4,18 @@ keywords: standards, compliance, security, benchmark
 title: CIS Kubernetes benchmark
 ---
 
-The Center for Internet Security (CIS) Kubernetes Benchmark is a reference document that can be used by system administrators, security and audit professionals and other IT roles in order to establish a secure configuration baseline for Kubernetes. CIS Benchmarks are developed by an open community of security practitioners and licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 Internal Public License](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode).
+The Center for Internet Security (CIS) Kubernetes Benchmark is a reference
+document that can be used by system administrators, security and audit
+professionals and other IT roles in order to establish a secure configuration
+baseline for Kubernetes. CIS Benchmarks are developed by an open community of
+security practitioners and licensed under a [Creative Commons
```

Comment:
style nit: Don't wrap links and things like that across lines. It can be hard to debug when Markdown breaks. I'd wrap this line after "a" and have the link text and target together on a single line even if it is more than 80 columns.